### PR TITLE
chore: remove zipkin in mcf-boilerplate [#174003427]

### PR DESCRIPTION
**Pull request by aundang**
[Story](https://www.pivotaltracker.com/n/projects/2432080)
[Slack Discussion](https://gahmen.slack.com/archives/GCW8M88M9/p1595561021000700)

This PR will remove zipkin from mcf-boilerplate.
An additional MR in api-job is also part of this chore:  https://gitlab.ci.mcf.sh/wsg/api-job/-/merge_requests/1593/diffs

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,8 +23,8 @@
     "chai": "4.1.2",
     "eslint": "4.19.1",
     "eslint-config-google": "0.9.1",
-    "lerna": "3.10.2",
-    "mocha": "5.2.0",
+    "lerna": "^3.22.1",
```
Comment (wynn987) [inline package.json:26]: can remove the ^ so it doesnt update without us knowing

File: package.json [hunk 2]
```diff
@@ -23,8 +23,8 @@
...
+    "mocha": "^8.1.2",
```
Comment (wynn987) [inline package.json:27]: same here

**Review by wynn987 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (aundang) [reply package.json]: already remove kindly checked.

File: package.json (hunk 2 above)
Comment (aundang) [reply package.json]: already removed. kindly checked

**Review by wynn987 — APPROVED** [review]

**Review by wynn987 — APPROVED** [review]

**Pull request merged**